.gitignore

```diff
@@ -21,3 +21,167 @@ xcuserdata/
 BeeSwift/Config.swift
 BeeSwift/GoogleService-Info.plist
 BeeSwift/Sentry.sh
+fastlane/test_output/*
+fastlane/report.xml
+
+
+# Created by https://www.toptal.com/developers/gitignore/api/fastlane,xcode,macos,swift,cocoapods
+# Edit at https://www.toptal.com/developers/gitignore?templates=fastlane,xcode,macos,swift,cocoapods
+
+### CocoaPods ###
+## CocoaPods GitIgnore Template
+
+# CocoaPods - Only use to conserve bandwidth / Save time on Pushing
+#           - Also handy if you have a large number of dependant pods
+#           - AS PER https://guides.cocoapods.org/using/using-cocoapods.html NEVER IGNORE THE LOCK FILE
+Pods/
+
+### fastlane ###
+# fastlane - A streamlined workflow tool for Cocoa deployment
+#
+# It is recommended to not store the screenshots in the git repo. Instead, use fastlane to re-generate the
+# screenshots whenever they are needed.
+# For more information about the recommended setup visit:
+# https://docs.fastlane.tools/best-practices/source-control/#source-control
+
+# fastlane specific
+fastlane/report.xml
+
+# deliver temporary files
+fastlane/Preview.html
+
+# snapshot generated screenshots
+fastlane/screenshots/**/*.png
+fastlane/screenshots/screenshots.html
+
+# scan temporary files
+fastlane/test_output
```

Comment:
configured to ignore this but files under this dir had become part of the repo in bad369544

how about we delete the generated report files (the ones being ignored here)?